websites/docs-smoke-test/src/content/views/code-examples.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: Code Examples
```

Comment:
I moved this in the "views" section. There is still a simpler version of code examples in the "components" pages, which we use for VRTs.